server/models/Transaction.js

```diff
@@ -632,7 +632,7 @@ function defineModel() {
     transaction,
     PaymentMethodId,
   }) => {
-    if (!transaction.amount) {
+    if (!transaction.amount && !transaction.platformTipInHostCurrency) {
```

Comment:
Sounds like we don't need this anymore?

```suggestion
    if (!transaction.amount) {
```